conf/bookkeeper.conf

```diff
@@ -23,6 +23,10 @@
 ## Server parameters
 #############################################################################
 
+# Bookie identifier, by default it is computed using the advertised bookies address
+# This value cannot be changed once the bookie is started for the first time
+#bookieId=
```

Comment:
We need clear documentation on how to use this and what is the path for migrating the existing cluster to use BookieId.

Reply:
sure, let's remove this line for the scope of this patch